src/main/java/tutoraid/logic/parser/ArgumentMultimap.java

```diff
@@ -35,9 +35,6 @@ public void put(Prefix prefix, String argValue) {
      * Returns the last value of {@code prefix}.
      */
     public Optional<String> getValue(Prefix prefix) {
-        if (!argMultimap.containsKey(prefix)) {
```

Comment:
Is this the one you mentioned not to delete?

Reply:
Reverting a change that shouldn't have been made to this method